### PR TITLE
fix: [ui] incorrect display of blank area at the bottom of the sidebar

### PR DESCRIPTION
**Pull request by pppanghu77**
fix incorrect display of blank area at the bottom of the sidebar and fix incorrect display of dividing lines at the bottom of the workspace

Log: fix: [ui] incorrect display of blank area at the bottom of the sidebar
Bug: https://pms.uniontech.com/bug-view-243531.html https://pms.uniontech.com/bug-view-241127.html https://pms.uniontech.com/bug-view-238993.html

### CODE REVIEW

**Review by GongHeng2017 — APPROVED** [review]

File: src/plugins/filemanager/core/dfmplugin-sidebar/treeviews/sidebarwidget.cpp [hunk 1]
```diff
@@ -305,21 +305,15 @@ void SideBarWidget::initializeUi()
...
     vlayout->addWidget(sidebarView);
     vlayout->setMargin(0);
     vlayout->setSpacing(0);
-    vlayout->addWidget(spacer);
 
     hlayout->addWidget(leftSpacer);
     hlayout->addLayout(vlayout);
 
     sidebarView->setModel(kSidebarModelIns.data());
+    kSidebarModelIns->addEmptyItem();
```
Comment (itsXuSt) [inline src/plugins/filemanager/core/dfmplugin-sidebar/treeviews/sidebarwidget.cpp:316]: 外部新增的侧边栏项可能出现在空白行的下面。

**Review by itsXuSt — CHANGES_REQUESTED** [review]
存在潜在 Bug

**Comment by pppanghu77** [comment]
> 存在潜在 Bug

已修复

**Review by max-lvs — APPROVED** [review]

**Pull request merged**